src/Api/DataProvider/AbstractElasticSearchDataProvider.php

```diff
@@ -107,21 +114,40 @@ protected function registerSearchNoHits(string $type, array $identifiers): void
      */
     protected function buildElasticQuery(string $type, array $identifiers): Query
     {
-        $boolQuery = new Query\BoolQuery();
+        $innerQuery = new Query\BoolQuery();
 
         foreach ($identifiers as $identifier) {
             $identifierFieldTermQuery = new Query\Term();
             $identifierFieldTermQuery->setTerm('isIdentifier', $identifier);
-            $boolQuery->addShould($identifierFieldTermQuery);
+            $innerQuery->addShould($identifierFieldTermQuery);
         }
 
+        $outerBoolQuery = new Query\BoolQuery();
+        $outerBoolQuery->addMust($innerQuery);
+
+        $range = new Query\Range();
+        $range->addField('height', [
+            'gte' => $this->params->get('elastic.min.image.size'),
```

Comment:
What if this has not been set?
Better to initialize the variable in the constructor in my opinion.

Reply:
Fixed